src/library/tabledelegates/bpmdelegate.cpp

```diff
@@ -50,6 +50,7 @@ BPMDelegate::BPMDelegate(QTableView* pTableView)
 #else
     m_pFactory->registerEditor(QVariant::Double, new BpmEditorCreator());
 #endif
+    m_cachedTextColor = QColor();
```

Comment:
Is this redundant? If not, please add a comment why it is reset here.

Reply:
Removed!